suite-native/intl/src/en.ts

```diff
@@ -66,6 +66,13 @@ export const en = {
             button: 'Steal session',
         },
     },
+    moduleDevUtils: {
```

Comment:
I would prefer not to translate devUtils since it's not meant to be accessible by users and it's only adding overhead

Reply:
If I'm not mistaken, the devUtils are displayed after the user clicks seven times on the commit hash. Since it is accessible by user (even though only advanced users), I believe it should be translated as well.